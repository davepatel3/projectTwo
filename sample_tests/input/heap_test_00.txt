1555
create_file
12
wNkt
0
print_layout
create_directory_root
P3-X
create_file
89
VZKW9PDYAQaDBB
2
get_available_size
get_absolute_path
3
create_file
20
lynXvJj5k19O_
2
create_directory
wcffJEh
0
exist
4
create_file
247
cMUpn4xm7
2
create_directory
LmY5_pVgG
2
create_file
13
gHRGqdTiH
0
create_file
140
Rqd-s8jOK8THWB0
7
create_directory
hBQ0PWsFT
7
get_file_size
6
create_directory
vq8uN9
5
create_directory
i9gEvpQdrNtr
11
get_file_size_path
/P3-X/LmY5_pVgG/Rqd-s8jOK8THWB0
create_file
66
aho1QemRKiUtzpd
5
create_file
14
BI8xosOK
11
create_directory
weErOlfngPcC5
10
create_directory
_fn-AakpFsMducodEwWv
10
print_layout
get_name
16
create_file
7
tDn0cGredwZHYW4F
10
create_directory
v8iD2 Bqr
10
create_file
61
pyxe0-edh_Q9
18
get_available_size
create_directory
6rE4It
7
create_directory
W9iTgJN6L5KzE w
15
get_file_size
1
get_absolute_path
7
print_layout
create_file
155
dGxin
7
create_directory
c1N8
15
create_directory
SXpQUXHVg2th5_v9fuhi
2
create_file
207
b_KCBl
10
create_directory
z86fn6U4lsL
5
get_handle
/P3-X/LmY5_pVgG/hBQ0PWsFT/_fn-AakpFsMducodEwWv
print_layout
create_directory
CqRBcf2yZ6TwJ
16
get_largest_file_handle
print_layout
create_file
255
9qqbj9KsQwaoBx
12
get_available_size
get_handle
/P3-X/SXpQUXHVg2th5_v9fuhi
print_layout
get_available_size
get_file_size_path
/P3-X/LmY5_pVgG/hBQ0PWsFT/b_KCBl
create_directory
vml9-vNeChtdeF
20
create_directory
DU5q
27
get_largest_file_handle
create_file
41
N7UqjIzE0E7iEg9g2RT0
26
create_file
17
PMrwHxo9gRzx -J3w
23
get_file_size_path
/P3-X/LmY5_pVgG/hBQ0PWsFT/tDn0cGredwZHYW4F
get_name
12
exist
9
get_largest_file_handle
get_file_size
3
create_directory
UKh5mld02yfw18Op
26
get_largest_file_handle
create_directory
MESmHpHn0r3e4YlO6nN
23
create_file
4
kq4f 18S S
34
create_directory
GTTfWG
12
get_available_size
get_file_size
1
create_file
60
F9J_ry1rR
21
create_directory
RLb1j0LBY2Ah36AOo0
20
create_directory
B9JqGXmnG8S1CanIaR
16
create_directory
VVxygBkMly
36
get_largest_file_handle
print_layout
get_available_size
create_file
25
 eTuNe6S3Pt
21
create_directory
mojEoHn sYTdv
40
create_directory
WsJrCf
23
create_directory
h28L
23
create_directory
tRhHlRpPIxdX
2
get_largest_file_handle
get_handle
/P3-X/LmY5_pVgG/hBQ0PWsFT/weErOlfngPcC5
get_name
12
get_available_size
get_absolute_path
15
get_available_size
create_directory
aQU2uWsbWEmV
27
create_file
70
H1vAobGbQnVBHflQ
23
get_available_size
get_name
5
create_directory
_CdA
36
create_directory
ki0PW
23
get_handle
/wcffJEh/vq8uN9/i9gEvpQdrNtr/9qqbj9KsQwaoBx
get_file_size
35
exist
37
create_file
24
tqH8Sp6
7
create_directory
kLj1eQ
42
get_absolute_path
38
create_directory
FsQsuU1yPQ
36
create_directory
4vnwFbwB_q
26
print_layout
get_file_size
50
create_file
52
O9OzGpOvY0IDOe4thvVb
44
create_directory
43HWPmdD
12
create_directory
F7YsmA2PI
46
create_directory
j1vpcV0LKB8wdSIob0
20
get_available_size
create_file
42
MQRVKqla1 _y
12
get_largest_file_handle
create_directory
5eDuDLBT
42
get_absolute_path
52
quit
