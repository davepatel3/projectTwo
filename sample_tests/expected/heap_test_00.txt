create_file
1
print_layout
[F]wNkt (size = 12)

create_directory_root
2
create_file
3
get_available_size
1454
get_absolute_path
/P3-X/VZKW9PDYAQaDBB
create_file
4
create_directory
5
exist
1
create_file
6
create_directory
7
create_file
8
create_file
9
create_directory
10
get_file_size
247
create_directory
11
create_directory
12
get_file_size_path
140
create_file
13
create_file
14
create_directory
15
create_directory
16
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
			[D]_fn-AakpFsMducodEwWv
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
[F]gHRGqdTiH (size = 13)

get_name
_fn-AakpFsMducodEwWv
create_file
17
create_directory
18
create_file
19
get_available_size
886
create_directory
20
create_directory
21
get_file_size
12
get_absolute_path
/P3-X/LmY5_pVgG
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
			[D]_fn-AakpFsMducodEwWv
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
		[D]6rE4It
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
[F]gHRGqdTiH (size = 13)

create_file
22
create_directory
23
create_directory
24
create_file
25
create_directory
26
get_handle
16
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
				[D]c1N8
			[D]_fn-AakpFsMducodEwWv
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
			[F]b_KCBl (size = 207)
		[D]6rE4It
		[F]dGxin (size = 155)
	[D]SXpQUXHVg2th5_v9fuhi
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
	[D]z86fn6U4lsL
[F]gHRGqdTiH (size = 13)

create_directory
27
get_largest_file_handle
6
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
				[D]c1N8
			[D]_fn-AakpFsMducodEwWv
				[D]CqRBcf2yZ6TwJ
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
			[F]b_KCBl (size = 207)
		[D]6rE4It
		[F]dGxin (size = 155)
	[D]SXpQUXHVg2th5_v9fuhi
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
	[D]z86fn6U4lsL
[F]gHRGqdTiH (size = 13)

create_file
28
get_available_size
269
get_handle
24
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
				[D]c1N8
			[D]_fn-AakpFsMducodEwWv
				[D]CqRBcf2yZ6TwJ
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
			[F]b_KCBl (size = 207)
		[D]6rE4It
		[F]dGxin (size = 155)
	[D]SXpQUXHVg2th5_v9fuhi
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
			[F]9qqbj9KsQwaoBx (size = 255)
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
	[D]z86fn6U4lsL
[F]gHRGqdTiH (size = 13)

get_available_size
269
get_file_size_path
207
create_directory
29
create_directory
30
get_largest_file_handle
28
create_file
31
create_file
32
get_file_size_path
7
get_name
i9gEvpQdrNtr
exist
1
get_largest_file_handle
28
get_file_size
89
create_directory
33
get_largest_file_handle
28
create_directory
34
create_file
35
create_directory
36
get_available_size
207
get_file_size
12
create_file
37
create_directory
38
create_directory
39
create_directory
40
get_largest_file_handle
28
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
					[F]F9J_ry1rR (size = 60)
				[D]c1N8
					[F]PMrwHxo9gRzx -J3w (size = 17)
					[D]MESmHpHn0r3e4YlO6nN
						[F]kq4f 18S S (size = 4)
			[D]_fn-AakpFsMducodEwWv
				[D]CqRBcf2yZ6TwJ
					[D]DU5q
				[D]B9JqGXmnG8S1CanIaR
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
			[F]b_KCBl (size = 207)
		[D]6rE4It
			[D]vml9-vNeChtdeF
			[D]RLb1j0LBY2Ah36AOo0
		[F]dGxin (size = 155)
	[D]SXpQUXHVg2th5_v9fuhi
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
			[F]9qqbj9KsQwaoBx (size = 255)
			[D]GTTfWG
				[D]VVxygBkMly
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
	[D]z86fn6U4lsL
		[F]N7UqjIzE0E7iEg9g2RT0 (size = 41)
		[D]UKh5mld02yfw18Op
[F]gHRGqdTiH (size = 13)

get_available_size
147
create_file
41
create_directory
42
create_directory
43
create_directory
44
create_directory
45
get_largest_file_handle
28
get_handle
15
get_name
i9gEvpQdrNtr
get_available_size
122
get_absolute_path
/P3-X/LmY5_pVgG/hBQ0PWsFT/weErOlfngPcC5
get_available_size
122
create_directory
46
create_file
47
get_available_size
52
get_name
wcffJEh
create_directory
48
create_directory
49
get_handle
28
get_file_size
4
exist
1
create_file
50
create_directory
51
get_absolute_path
/P3-X/LmY5_pVgG/6rE4It/RLb1j0LBY2Ah36AOo0
create_directory
52
create_directory
53
print_layout
[F]wNkt (size = 12)
[D]P3-X
	[F]VZKW9PDYAQaDBB (size = 89)
	[F]lynXvJj5k19O_ (size = 20)
	[F]cMUpn4xm7 (size = 247)
	[D]LmY5_pVgG
		[F]Rqd-s8jOK8THWB0 (size = 140)
		[D]hBQ0PWsFT
			[D]weErOlfngPcC5
				[D]W9iTgJN6L5KzE w
					[F]F9J_ry1rR (size = 60)
					[F] eTuNe6S3Pt (size = 25)
				[D]c1N8
					[F]PMrwHxo9gRzx -J3w (size = 17)
					[D]MESmHpHn0r3e4YlO6nN
						[F]kq4f 18S S (size = 4)
					[D]WsJrCf
					[D]h28L
					[F]H1vAobGbQnVBHflQ (size = 70)
					[D]ki0PW
			[D]_fn-AakpFsMducodEwWv
				[D]CqRBcf2yZ6TwJ
					[D]DU5q
					[D]aQU2uWsbWEmV
				[D]B9JqGXmnG8S1CanIaR
			[F]tDn0cGredwZHYW4F (size = 7)
			[D]v8iD2 Bqr
				[F]pyxe0-edh_Q9 (size = 61)
			[F]b_KCBl (size = 207)
		[D]6rE4It
			[D]vml9-vNeChtdeF
			[D]RLb1j0LBY2Ah36AOo0
		[F]dGxin (size = 155)
		[F]tqH8Sp6 (size = 24)
	[D]SXpQUXHVg2th5_v9fuhi
	[D]tRhHlRpPIxdX
[D]wcffJEh
	[D]vq8uN9
		[D]i9gEvpQdrNtr
			[F]9qqbj9KsQwaoBx (size = 255)
			[D]GTTfWG
				[D]VVxygBkMly
					[D]mojEoHn sYTdv
						[D]kLj1eQ
				[D]_CdA
				[D]FsQsuU1yPQ
		[F]BI8xosOK (size = 14)
	[F]aho1QemRKiUtzpd (size = 66)
	[D]z86fn6U4lsL
		[F]N7UqjIzE0E7iEg9g2RT0 (size = 41)
		[D]UKh5mld02yfw18Op
		[D]4vnwFbwB_q
[F]gHRGqdTiH (size = 13)

get_file_size
24
create_file
Exceeds file system size limit!
create_directory
54
create_directory
55
create_directory
56
get_available_size
28
create_file
Exceeds file system size limit!
get_largest_file_handle
28
create_directory
57
get_absolute_path
/wcffJEh/vq8uN9/i9gEvpQdrNtr/GTTfWG/FsQsuU1yPQ
quit
